command.go

```diff
@@ -146,8 +146,15 @@ func isZero(v reflect.Value) bool {
 		// Types that are not allowed at all.
 		// NOTE: Would be better with its own error for this.
 		return true
-	case reflect.Map, reflect.Array, reflect.Slice:
+	case reflect.Map, reflect.Slice:
 		return v.IsNil()
+	case reflect.Array:
```

Comment:
Could you add a test case that triggers the undesired behaviour (which this code then makes the test pass)?